tests/backup_helper.go

```diff
@@ -8752,3 +8752,59 @@ func ValidatePVCCleanup(pvcBefore, pvcAfter []string) error {
 
 	return nil
 }
+
+type MaintenanceJobType string
+
+const (
+	FullMaintenanceJob  MaintenanceJobType = "full-maintenance-repo"
+	QuickMaintenanceJob MaintenanceJobType = "quick-maintenance-repo"
+)
+
+// ModifyMaintenanceJobFrequency modifies the frequency of the cron job
+func ModifyMaintenanceJobFrequency(backupLocationName string, frequencyInMinutes int, maintenanceJobType MaintenanceJobType) error {
+	k8sBatch := batch.Instance()
+	cronJob, err := GetCronJobByBackupLocation(backupLocationName, maintenanceJobType)
+	if err != nil {
+		return err
+	}
+	cronJob.Spec.Schedule = fmt.Sprintf("*/%d * * * *", frequencyInMinutes)
```

Comment:
Should we support all the cron params as it will be helpful later in case of longevity and scale test frameworks?

Reply:
In the next PR I will be providing a function specific to update the image and memory of the cron job.